packages/field-base/test/input-constraints-mixin.test.js

```diff
@@ -65,11 +65,34 @@ const runTests = (baseClass) => {
         expect(validateSpy.called).to.be.false;
       });
 
-      it('should not validate when setting a constraint', async () => {
+      it('should not validate on setting a constraint', async () => {
         element.required = true;
         await nextFrame();
         expect(validateSpy.called).to.be.false;
       });
+
+      it('should not validate on removing a not last constraint when the field is valid', async () => {
+        element.pattern = '\\d*';
+        element.minlength = 2;
+        await nextFrame();
+        expect(element.invalid).to.be.false;
+        validateSpy.resetHistory();
+        element.minlength = 2;
+        await nextFrame();
+        expect(validateSpy.called).to.be.false;
+      });
+
+      it('should validate on removing a not last constraint when the field is invalid', async () => {
```

Comment:
_a not last constraint_ doesn't sound nice, maybe other suggestions?

Reply:
Would _"Should not validate an invalid field on a non-final constraint removal"_ sound right?

Reply:
`non-final` sounds good!  I would prefer `on removing` though to keep it consistent with other tests. Replaced `not last ` with `non-final`.